src/csharp/Grpc.Core.Xamarin/Grpc.Core.Xamarin.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\Grpc.Core\Common.csproj.include" />
+
+  <PropertyGroup>
+    <Authors>The gRPC Authors</Authors>
+    <Copyright>Copyright 2015 The gRPC Authors</Copyright>
+    <Description>Xamarin support for gRPC C#. Note that the gRPC C# support for the Xamarin platform is considered experimental.
+    This package contains the native grpc_csharp_ext libraries that are needed to run gRPC C# on mobile platforms (Android, iOS).
+    WARNING: Versions of Grpc.Core.Xamarin and Grpc.Core dependencies being used in your project must always match exactly, otherwise
```

Comment:
We could add this "exact version" constraint in a .nuspec file (which is the "old" way of specifying the metadata for a nuget package), but for consistency reasons, I'd like to build the nuget  from a .csproj (same as all our other nugets).

The .csproj's `<ProjectReference>` unfortunately doesn't allow specifying the need for exact version match and introducing .nuspec is messy / clumsy (as the csproj and nuspec don't really mix that well).

Reply:
Can we not specify a PackageReference with a constraint of (say, for 2.34) `[2.34.0, 2.35.0)`, maybe in some conditional way so we could just use the ProjectReference when building locally? I understand that would require the Grpc.Core package to be published (e.g. to a local nuget source) before building this one, but it would achieve the desired effect. I do agree with avoiding bringing NuSpec into it.

Reply:
I can try to experiment with PackageReference vs ProjectReference a bit more, but this also sounds like a hack. Currently all our nuget packages are built by a single invocation of a script, so making sure that Grpc.Core package is built and then putting it into a local nuget feed (so that it is "published") and only then building Grpc.Core.Xamarin seems ugly and complicated, so I'd like to avoid it if possible.